corehq/apps/reports/static/reports/js/edit_scheduled_report.js

```diff
@@ -49,6 +52,13 @@ hqDefine("reports/js/edit_scheduled_report", [
                 $('[name="interval"]').change(function () {
                     update_day_input(self.weekly_options, self.monthly_options);
                 });
+                $("#id_start_date").datepicker({
+                    dateFormat: "yy-mm-dd",
+                    minDate: 0,
```

Comment:
Shouldn't `minDate` be today?

Reply:
I thought that's what I was doing here (http://jqueryui.com/datepicker/#min-max). Is there a more accurate/better way to set `minDate` to the current date?

Reply:
Ha, perhaps I should have read the docs. I didn't know minDate/maxData could be offsets.